test/galaxy_selenium/has_driver.py

```diff
@@ -14,6 +15,7 @@
 
 
 class HasDriver:
+    TimeoutException = TimeoutException
```

Comment:
I guess because it's used as `self.TimeoutException` in `NavigatesGalaxy` subclass, sorry for the noise